src/core/server.ts

```diff
@@ -13,30 +14,36 @@ interface MockServerConfig {
 }
 
 class MockServer {
-    config: MockServerConfig;
-    configFetcher: IConfigFetcher;
+    mockConfig: MockServerConfig;
+    config: IConfig;
```

Comment:
These both can be merged into 1 as both are same.

```
mockServerConfig: MockServerConfig
```

Also IConfig is incorrect name as prefix `I` is used for marking abstract classes

Reply:
This can be converted to this
`constructor (config: MockServerConfig)`